client/wfm/src/model/WorkOrderResult.ts

```diff
@@ -5,9 +5,6 @@ import { StepResult } from './StepResult';
  * On-going progression of a {@link WorkOrder}, contains data for the individual {@link Step}s
  */
 export interface WorkOrderResult {
```

Comment:
Just get rid of this interface entirely

Reply:
Yes - I'm planning to but I'm working different way here. 
I split my work into phases where I do minimal change and then test entire solution.
This phases should be relatively small etc.

Reply:
:+1: was just pointing out from the previous comment